DBManager.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import create_engine
+
+
+class DBManager:
```

Comment:
please review the code that we have already written - we have a file called `db_utils.py` that does exactly this and it is what you should be using